### PR TITLE
feature: add popupp

### PR DESCRIPTION
**Pull request by Regiss05**
This feature should:

- add popupp

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -94,7 +97,55 @@ <h3>Multi-Post Stories</h3>
                             <li>Ruby</li>
                         </ul>
                     </div>
-                    <button class="btn btn-style" type="button">See Project</button>
+                    <button data-modal-target="#modal" class="btn btn-style" type="button">See Project</button>
+                    <div class="modal" id="modal">
```
Comment (Lordkaito) [inline index.html:101]: - [ ] Good job applying the modal here, but please make sure to make it responsive so you can see the modal properly in both mobile and desktop versions. Please apply the necessary changes.

File: index.html [hunk 2]
```diff
@@ -94,7 +97,55 @@ <h3>Multi-Post Stories</h3>
...
+                                            <span>See Source</span>
+                                            <span class="icon-btn">
+                                                <ion-icon name="logo-github"></ion-icon>
+                                            </span>
+                                        </button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
```
Comment (Lordkaito) [inline index.html:147]: - [ ] Here, you just hardcoded all the elements inside your modal, but you were supposed to use a JavaScript array or object to store all the elements of your modal and make it appear dynamically once the page loads. So please, make sure to apply the necessary changes.

File: index.html [hunk 3]
```diff
@@ -94,7 +97,55 @@ <h3>Multi-Post Stories</h3>
...
+                    <div id="overlay"></div>
                 </div>
             </section>
             <div class="container-form">
```
Comment (Lordkaito) [inline index.html:151]: - [ ] Here happens the same as I mentioned before, you now are supposed to store all your elements of this `div` inside a JavaScript object and make the elements appear dynamically once the page loads, so try to apply the changes here as well.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Regiss05, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Regiss05 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!You can tag me using @AnselemOdims for a quicker response or reach out on slack.

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @Regiss05, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**